.github/workflows/pr_build.yaml

```
@@ -239,10 +239,9 @@ jobs:
     outputs:
       test: ${{ steps.set-matrix.outputs.test }}
   
-
   integration:
-    name: integration (linux)
-    runs-on: ubuntu-22.04
+    name: integration (${{ matrix.arch }})
```

One small nit I have for this is that the name of the integration steps have lost the job index:
```
- PR Build / integration (linux) (5, 4)
+ PR Build / integration (x64)
```

I wonder if that's something that we can bring back, maybe something like:
```suggestion
    name: integration (${{ matrix.arch }}) (${{ strategy.job-index}}/${{ strategy.job-total }})
```